x/stake/types/validator.go

```diff
@@ -85,14 +94,14 @@ func NewSideChainValidator(feeAddr sdk.AccAddress, operator sdk.ValAddress, desc
 		UnbondingMinTime:   time.Unix(0, 0).UTC(),
 		UnbondingHeight:    int64(0),
 		Commission:         NewCommission(sdk.ZeroDec(), sdk.ZeroDec(), sdk.ZeroDec()),
-		DistributionAddr:   generateDistributionAddr(operator, sideChainId),
+		DistributionAddr:   GenerateDistributionAddr(operator, sideChainId),
 		SideChainId:        sideChainId,
 		SideConsAddr:       sideConsAddr,
 		SideFeeAddr:        sideFeeAddr,
 	}
 }
 
-func generateDistributionAddr(operator sdk.ValAddress, sideChainId string) sdk.AccAddress {
+func GenerateDistributionAddr(operator sdk.ValAddress, sideChainId string) sdk.AccAddress {
```

Comment:
TODO: implement the white-label oracle relayers.

Reply:
done